src/xenia/app/xenia_main.cc

```diff
@@ -278,27 +278,22 @@ int xenia_main(const std::vector<std::string>& args) {
   }
 
   if (cvars::mount_cache) {
-    auto cache0_device =
-        std::make_unique<xe::vfs::HostPathDevice>("\\CACHE0", "cache0", false);
-    if (!cache0_device->Initialize()) {
-      XELOGE("Unable to scan cache0 path");
-    } else {
-      if (!emulator->file_system()->RegisterDevice(std::move(cache0_device))) {
-        XELOGE("Unable to register cache0 path");
-      } else {
-        emulator->file_system()->RegisterSymbolicLink("cache0:", "\\CACHE0");
-      }
-    }
+    const std::pair<std::string, std::string> cache_mounts[] = {
```

Comment:
std::string (instead of const char*) is discouraged for literals due to potential dynamic allocation (though not likely to happen in practice here specifically due to small string optimization, but in the general case it's possible).

Reply:
I would go with `const std::string_view`. The used APIs expect that anyways. Playing with godbolt I found out clang 11 will call `strlen` on a `const char*` even with `-O3`. That is scary...

This isn't hot code though so we shouldnt get too crazy...